EduNLP/SIF/sif.py

```diff
@@ -11,6 +11,8 @@
 def is_sif(item):
     itemparser = Parser(item)
     itemparser.description_list()
+    if itemparser.fomula_illegal_flag:
+        raise Exception(itemparser.fomula_illegal_message)
```

Comment:
Instead of using Exception, use some more concrete exception types, like ValueError, TypeError and so on